test/add/sdk/sdk_test.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS d.file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:path/path.dart' as p;
+
+import 'package:test/test.dart';
+
+import '../../descriptor.dart' as d;
+import '../../test_pub.dart';
+
+void main() {
+  setUp(() async {
+    await servePackages((builder) {
+      builder.serve('bar', '1.0.0');
+    });
+
+    await d.dir('flutter', [
+      d.dir('packages', [
+        d.dir('foo', [
+          d.libDir('foo', 'foo 0.0.1'),
+          d.libPubspec('foo', '0.0.1', deps: {'bar': 'any'})
+        ])
+      ]),
+      d.dir('bin/cache/pkg', [
+        d.dir(
+            'baz', [d.libDir('baz', 'foo 0.0.1'), d.libPubspec('baz', '0.0.1')])
+      ]),
+      d.file('version', '1.2.3')
+    ]).create();
+  });
+
+  test("gets an SDK dependency's dependencies", () async {
```

Comment:
```suggestion
  test("adds an SDK dependency's dependencies", () async {
```

And same changes to other test cases :)

I assume these are more less copied from some `pub get` tests.

Reply:
haha whoops